samples/logging-sample/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/GoogleCloudPlatform/cloud-build-notifiers/samples/logging-sample
```

Comment:
Up to you: add copyright header here too?

Reply:
I think it's OK without -- I checked our [golang-samples](https://github.com/GoogleCloudPlatform/golang-samples) repo and spot checked a few go.mod's and didn't see any copyright headers on them.